tests/purs/bundle/3551/ModuleWithDeadCode.js

```diff
@@ -1,10 +1,8 @@
-"use strict";
-
-var fs = require('fs');
+import * as fs from 'fs';
 
 var source = fs.readFileSync(__filename, 'utf-8');
```

Comment:
This works because the output of `purs bundle` is interpreted as a CommonJS module by Node.js but `__filename` is not available in ES modules, so we’ll have to support CommonJS foreign modules at least until `language-javascript` can parse `import.meta.url`.